color-picker-project/src/components/colorpicker/ColorPicker.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, Fragment } from 'react';
+import { hsvaToHex } from '@uiw/color-convert';
+import { hsvaToRgbaString } from '@uiw/color-convert';
+import Wheel from '@uiw/react-color-wheel';
+import ShadeSlider from '@uiw/react-color-shade-slider';
+import Alpha from '@uiw/react-color-alpha';
+import Material from '@uiw/react-color-material';
+import EditableInputHSLA from '@uiw/react-color-editable-input-hsla';
+import './ColorPicker.css'
+
+function ColorPicker() {
+
+  const [hsva, setHsva] = useState({ h: 214, s: 43, v: 90, a: 1 });
+
+  return (
+    <div className="color-picker-div">
+      <Fragment>
+        <div className="color-picker-container">
+          <h1 style= {{ color: hsvaToHex(hsva) }} className="color-picker-title">Choose a color...</h1>
+          <Wheel className="color-picker-wheel" color={hsva} onChange={(color) => setHsva({ ...hsva, ...color.hsva })} />
+          <ShadeSlider
+          hsva={hsva}
+          style={{ width: 210, marginTop: 20 }}
+          onChange={(newShade) => {
+            setHsva({ ...hsva, ...newShade });
+          }}
+          />
+          <Material className="color-picker-material" style={{ width: '15%' }}
```

Comment:
This needs some debugging.  The first thing I went to do when I ran this in local host was type "FFF" into the hex input.  The input doesn't let you type more than two characters.  I couldn't edit the RGB inputs at all.

I'm assuming these inputs are controlled by the color wheel only, and are therefore just displaying values.  But I would argue there needs to be a way for the user to just type in a code and that color gets selected on the color wheel.  

See Google's color picker as an example.

Reply:
Yeah, this only takes the color from the wheel. I can change that if we want to.